src/Week.jsx

```diff
@@ -97,6 +97,7 @@ export default class Week extends PureComponent {
           className={styles.header}
           style={{
             paddingLeft: RULER_WIDTH_IN_PIXELS,
+            paddingRight: '15px',
```

Comment:
No big deal, but this could be in the css file. I try to use inline styles only for stuff that needs a js variable or a constant (like the paddingLeft above).